### PR TITLE
Fix Null value handling for Min, Max, and Value in AdaptiveNumberInput

### PR DESCRIPTION
**Pull request by MattSFT**
Fix with tests to this issue: https://github.com/Microsoft/AdaptiveCards/issues/2478

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards/AdaptiveNumberInput.cs [hunk 1]
```diff
@@ -33,7 +33,8 @@ public class AdaptiveNumberInput : AdaptiveInput
 
         /// <summary>
         ///     The initial value for the field
-        /// </summary>
+        /// </summary>'
```
Comment (paulcam206) [inline source/dotnet/Library/AdaptiveCards/AdaptiveNumberInput.cs]: >' [](start = 22, length = 1)

typo :)

Comment (MattSFT) [reply source/dotnet/Library/AdaptiveCards/AdaptiveNumberInput.cs]: fixed

**Comment by MattSFT** [comment]
@paulcam206 I don't have permission to merge this in. Could I get some help?

**Comment by paulcam206** [comment]
Done. 

Thanks! :)

**Pull request merged**